packages/vm/lib/evm/opcodes.ts

```diff
@@ -36,198 +36,194 @@ export interface OpcodeList {
 }
 
 // Base opcode list. The opcode list is extended in future hardforks
-const opcodes: OpcodeList = createOpcodes({
+const opcodes = {
```

Comment:
can later on  be renamed to `chainstartOpcodes` for clarification (after support for other old HFs is added)